tests/histogram/plotter_TestCase.py

```diff
@@ -159,7 +166,8 @@ def pysuite():
 
 def main():
     global interactive
-    interactive = True
+    # interactive = True
+    interactive = False
```

Comment:
was the  interactive = True creating an issue somewhere? Can you include the information about this change in your PR description please?

Reply:
This can be set back to True. It was just annoying that it keeps popping up pyplot window when interactive to set true. Forgot to change it back. So now it's back to original form.

Reply:
is this back to True?